python/pipeline/stack.py

```diff
@@ -408,7 +407,7 @@ def _make_tuples(self, key):
             f = performance.parallel_motion_stack # function to map
             raster_phase = (RasterCorrection() & key).fetch1('raster_phase')
             fill_fraction = (StackInfo() & key).fetch1('fill_fraction')
-            max_y_shift, max_x_shift = 20 / (StackInfo.ROI() & key).microns_per_pixel
+            max_y_shift, max_x_shift = 20 / (StackInfo.ROI() & key).microns_per_pixel[1:]
```

Comment:
it's just a threshold I choose for max shift (by asking Jake and I only use the second axes on, because the first axes is depth/z and we only have shifts in y and x